### PR TITLE
[v5] types/jsdoc cleanup

### PR DESCRIPTION
**Pull request by MananTank**
## PR-Codex overview
This PR updates import paths and improves wallet management in `thirdweb`. 

### Detailed summary
- Updated import paths for `uuid`, `encodeJWT`, `refreshJWT`, `decodeJWT`
- Refactored wallet connection methods
- Added `WalletEmitterEvents` type
- Improved wallet creation and connection options
- Updated `ConnectWallet` component props and default wallets list

> The following files were skipped due to too many changes: `packages/thirdweb/src/react/web/ui/ConnectWallet/ConnectWalletProps.ts`, `packages/thirdweb/src/react/core/hooks/connection/useAutoConnect.tsx`, `packages/thirdweb/src/wallets/wallet-types.ts`, `packages/thirdweb/src/react/web/ui/ConnectWallet/Modal/ConnectEmbed.tsx`

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: packages/thirdweb/src/adapters/ethers5.ts [hunk 1]
```diff
@@ -71,7 +71,7 @@ export const ethers5Adapter = /* @__PURE__ */ (() => {
        * @returns A Promise that resolves to an ethers.js Contract.
        * @example
        * ```ts
-       * import { ethers5Adapter } from "@thirdweb/adapters";
+       * import { ethers5Adapter } from "thirdweb/adapters";
```
Comment (jnsdls) [inline packages/thirdweb/src/adapters/ethers5.ts]:
```suggestion
       * import { ethers5Adapter } from "thirdweb/adapters/ethers5";
```

File: packages/thirdweb/src/react/web/ui/ConnectWallet/ConnectWalletProps.ts [hunk 2]
```diff
@@ -302,7 +302,7 @@ export type ConnectButtonProps = {
    * import { createWallet, embeddedWallet } from "thirdweb/react";
```
Comment (jnsdls) [inline packages/thirdweb/src/react/web/ui/ConnectWallet/ConnectWalletProps.ts:302]:
```suggestion
   * import { createWallet, embeddedWallet } from "thirdweb/wallets";
```

File: packages/thirdweb/src/react/web/ui/ConnectWallet/ConnectWalletProps.ts [hunk 3]
```diff
@@ -302,7 +302,7 @@ export type ConnectButtonProps = {
...
    *
    * const wallets = [
-   *   embeddedWallet(),
+   *   createWallet("embedded"),
```
Comment (jnsdls) [inline packages/thirdweb/src/react/web/ui/ConnectWallet/ConnectWalletProps.ts:305]:
```suggestion
   *   embeddedWallet(),
```

File: packages/thirdweb/src/react/web/ui/ConnectWallet/Modal/ConnectEmbed.tsx [hunk 4]
```diff
@@ -495,16 +410,16 @@ const ConnectEmbedContent = (
...
-  useEffect(() => {
-    if (requiresSignIn && screen === reservedScreens.main) {
-      setScreen(reservedScreens.signIn);
-    }
-  }, [requiresSignIn, screen, setScreen]);
+  // useEffect(() => {
+  //   if (requiresSignIn && screen === reservedScreens.main) {
+  //     setScreen(reservedScreens.signIn);
+  //   }
+  // }, [requiresSignIn, screen, setScreen]);
```
Comment (jnsdls) [inline packages/thirdweb/src/react/web/ui/ConnectWallet/Modal/ConnectEmbed.tsx:422]: maybe just delete, can bring it back later?

**Review by jnsdls — CHANGES_REQUESTED** [review]
couple things

File: packages/thirdweb/src/adapters/ethers5.ts (hunk 1 above)
Comment (MananTank) [reply packages/thirdweb/src/adapters/ethers5.ts]: just fixed

**Review by jnsdls — APPROVED** [review]
LGTM

**Pull request merged**